### ISSUE TITLE
Images lacking alt text

### ISSUE THREAD

**Issue by mattl** [creativecommons/new-creativecommons.org#25]
The logo in the main banner is lacking alt text. The logo in the footer has alt text but it's just the filename of the image.

**Comment by fmann** [comment]
I've added alt text to the header logo.

**Comment by mattl** [comment]
Great. The footer logo is still kinda funky.

**Comment by mattl** [comment]
I fixed up the alt text on the platforms page @janeatcc -- if there are other places on the site with all the various platforms linked, the markup at https://staging.creativecommons.org/about/platform/ is pretty solid now.

**Comment by ageeroms** [comment]
I copied your updated code to
https://staging.creativecommons.org/about/platform/ - thanks!



On Tue, Jun 21, 2016 at 6:40 PM, Matt Lee <notifications@github.com> wrote:

> I fixed up the alt text on the platforms page @janeatcc
> <https://github.com/janeatcc> -- if there are other places on the site
> with all the various platforms linked, the markup at
> https://staging.creativecommons.org/about/platform/ is pretty solid now.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/creativecommons/new-creativecommons.org/issues/25#issuecomment-227620718>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGXZDlejrBEtOogIjTSqRLxnImugMUE3ks5qOJKkgaJpZM4I7OQZ>
> .
>



-- 
Anna Geeroms | Business Analyst
Affinity Bridge

http://affinitybridge.com
T. 604.569.0510 | E. anna@affinitybridge.com

710 - 207 W. Hastings St.
Vancouver, BC
V6B 1H7
